components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -130,11 +133,50 @@ WithForcedColors.parameters = {
 export const Disabled = Template.bind({});
 Disabled.tags = ["!autodocs", "!dev", "test"];
 Disabled.args = {
-	isDisabled: true
+	isDisabled: true,
 };
 
 export const Quiet = Template.bind({});
 Quiet.tags = ["!autodocs", "!dev", "test"];
```

Comment:
I think I asked this on the autodocs config PR, but why are these Quiet and Disabled stories removed from the docs page? They're on the docs site right now, but are we "saving" them for just VRT? Should we be doing that for other components as well?

Reply:
I've added these to the docs page.